packages/app/metrics-utils/package.json

```diff
@@ -0,0 +1,50 @@
+{
+    "name": "@lokalise/metrics-utils",
+    "version": "v1.0.0",
+    "files": ["dist"],
+    "license": "Apache-2.0",
+    "type": "module",
+    "main": "./dist/index.cjs",
+    "module": "./dist/index.js",
+    "types": "./dist/index.d.ts",
+    "homepage": "https://github.com/lokalise/shared-ts-libs",
+    "repository": {
+        "type": "git",
+        "url": "git://github.com/lokalise/shared-ts-libs.git"
+    },
+    "exports": {
+        ".": {
+            "types": "./dist/index.d.ts",
+            "require": "./dist/index.cjs",
+            "import": "./dist/index.js"
+        }
+    },
+    "publishConfig": {
+        "access": "public"
+    },
+    "scripts": {
+        "build": "rimraf dist && vite build",
+        "dev": "vite watch",
+        "clean": "rimraf dist",
+        "lint": "biome check . && tsc --noEmit",
+        "lint:fix": "biome check --write",
+        "test": "vitest run",
+        "test:ci": "vitest run --coverage"
+    },
+    "peerDependencies": {
+        "prom-client": "^15.1.3",
+        "fastify-metrics": "^12.1.0"
+    },
+    "dependencies": {
+        "@lokalise/fastify-extras": "^24.3.1"
+    },
+    "devDependencies": {
+        "@biomejs/biome": "^1.9.2",
+        "@lokalise/biome-config": "^1.5.0",
+        "@lokalise/package-vite-config": "3.1.0",
+        "@vitest/coverage-v8": "^2.1.1",
+        "rimraf": "^6.0.1",
+        "typescript": "5.6.2",
+        "vitest": "^2.0.3"
```

Comment:
let's use latest vitest

Reply:
Updated `vitest` and `vite` to latest stable versions ✅